### PR TITLE
SC-13524 Simplify Readme and link to official docs

### PR DESCRIPTION
**Pull request by PabloB94**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,80 +2,40 @@
 
 The Sematext Operator for Kubernetes provides an easy way to deploy Sematext Agent.
 
-Sematext Agent collects metrics about hosts (CPU, memory, disk, network, processes), containers (both Docker and rkt) and orchestrator platforms and ships that to [Sematext Cloud](https://sematext.com/cloud). Sematext Cloud is available in the US and EU regions.
+Sematext Agent collects metrics about hosts (CPU, memory, disk, network, processes), containers (both Docker and rkt) and orchestrator platforms, and ships that to [Sematext Cloud](https://sematext.com/cloud). Sematext Cloud is available in the US and EU regions.
```
Comment (otisg) [inline README.md]: And containerd?  And podman?

File: README.md [hunk 2]
```diff
@@ -2,80 +2,40 @@
...
 
 It installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
 
 ## Quickstart
 
+You need to create [an Infra App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create/app/infra) to get your infra token.
```
Comment (otisg) [inline README.md]: + EU link?
Infra App token

File: README.md [hunk 3]
```diff
@@ -2,80 +2,40 @@
...
 
 Run the `kubectl delete` command to remove the operator and its dependencies:
 
 ```sh
 kubectl delete -f https://raw.githubusercontent.com/sematext/sematext-operator/master/bundle.yaml
 ```
+
+## Docs
+
+For more details refer to our [official documentation](https://sematext.com/docs/agents/sematext-agent/kubernetes/operator/).
```
Comment (otisg) [inline README.md]: Same comment as in the other PR

**Review by otisg — COMMENTED** [review]
Minor mods suggested

File: README.md (hunk 1 above)
Comment (yagogou) [reply README.md]: +1

File: README.md [hunk 4]
```diff
@@ -2,80 +2,40 @@
...
+Once installed, you can create `SematextAgent` resource that deploys Sematext Agent to all nodes in your cluster via `DaemonSet` resource:
 
 ```yaml
 apiVersion: sematext.com/v1alpha1
 kind: SematextAgent
 metadata:
   name: test-sematextagent
 spec:
-  region: "EU"
-  containerToken: YOUR_CONTAINER_TOKEN
```
Comment (yagogou) [inline README.md:56]: I would test this before removing it, not sure how it works the operator or which token use

Comment (PabloB94) [reply README.md:56]: The operator uses the Helm chart, so it has the same parameters. This token does not exist anymore

**Review by otisg — APPROVED** [review]
LGTM

**Pull request merged**